init-net.sh

```diff
@@ -0,0 +1,9 @@
+#!/vendor/bin/sh
+
+
+if [ ! -d /sys/class/net/wlan0 ]; then
+	/system/bin/ifconfig eth0 down
+	/system/bin/ip link set eth0 name wifi_eth
+	/system/bin/ifconfig wifi_eth up
+	/system/bin/ip link add link wifi_eth name wlan0 type virt_wifi
+fi
```

Comment:
This needs rework.
1. Do **NOT** rename `eth0` like that, people will not be happy to have their main ethernet connection be renamed like that.
    - Not to mention that systems can have different names instead of `eth0`.
Most distros have nowerdays changed to use `eno1` as default for example...
1. Not all systems have their wifi named `wlan0`.
It's better to find an automated way of finding what name the wifi card(s) use.
Maybe something like:
   ```sh
   wificards=( $(iwconfig 2>/dev/null | grep 'IEEE 802.11' | cut -d ' ' -f 1) )
   for wifi in "${wificards[@]}"; do
    ...
   done
   ```

Last but not least, i would prefer it much more to use [netdev](https://www.freedesktop.org/software/systemd/man/latest/systemd.netdev.html) files to setup this `virt_wifi` link.